### DISCUSSION TITLE
Take static png/jpg map and convert to .map file to be used with mapsforge

### DISCUSSION THREAD

**Discussion by tomjarvela** [mapsforge/mapsforge#1390] [Q&A] [answered]
Hi!

I have a local trail system that my family uses often. I would like to be able to take a png of that map system, geo reference it, and convert it to a .map file to be able to utilize it with mapsforge.

Is this possible?
If so, is there anybody who would be kind enough to walk me through the basics steps to accomplish this?

I've taken a look at Mapsforge Map-Writer to convert an OSM into maps that can be used. But I want to start with a static image file.

I am not against having to jump through a few services to learn how to do this.

Thanks in advance!
Best,
Tom

**Comment by devemux86** [comment, marked as answer]
Mapsforge map files are created from (and contain) vector data, not images.

What you mean is probably offline image tiles.
See the [TileStoreLayerViewer](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-samples-android/src/main/java/org/mapsforge/samples/android/TileStoreLayerViewer.java) for a reader example.

Creation of image tiles can be done with several ways (better search online).
With [Mobile Atlas Creator](https://mobac.sourceforge.io/) or [QGIS](https://www.qgis.org/) or command line tools like [GDAL](https://gdal.org/).
